waisntechtools/alerts/Twilio/fetch_sms.py

```diff
@@ -0,0 +1,29 @@
+from twilio.rest import Client
+from django.conf import settings
+from django.http import HttpResponseForbidden
+from twilio.request_validator import RequestValidator
+
+import requests
+import os
+
+
+def fetch_twilio(request):
+    account_sid = os.environ.get('TWILIO_ACCOUNT_SID')
+    auth_token = os.environ.get('TWILIO_AUTH_TOKEN')
+
+    validator = RequestValidator(auth_token)
+    request_valid = validator.validate(
+        request.build_absolute_uri(),
+        request.POST,
+        request.META.get('HTTP_X_TWILIO_SIGNATURE', ''))
+
+    if request_valid or settings.DEBUG:
+        client = Client(account_sid, auth_token)
+        message_fetch = client.messages.list(
+            to='+13174899826')
+        messages = []
+        for message in message_fetch:
```

Comment:
What is the reason that `message_fetch` can't be returned as is? It looks like the code is just creating a copy of it.

Reply:
I don't remember why I had put this in there but I just removed it and it seems to work fine